pkg/apis/core/validation/validation.go

```diff
@@ -1638,6 +1639,14 @@ var allowedPVCTemplateObjectMetaFields = map[string]bool{
 	"Labels":      true,
 }
 
+// PersistentVolumeValidationOptions contains the different settings for PeristentVolume validation
+type PersistentVolumeValidationOptions struct {
+	// Allow validation of inline PersistentVolume specs
+	ValidateInlinePersistentVolumeSpec bool
```

Comment:
I think we should keep this structure focused on migrations (safely adding new fields/values).

It looks like `ValidateInlinePersistentVolumeSpec` is not about this, but a different usage of the same API. Did I get this right? If so, please remove this field from here.

I would actually prefer to have to separate functions for `ValidatePersistentVolumeSpec` with/without inline. But that's out of the scope of this PR.

Reply:
Yeah that makes sense. Removed the field and re-added the bool arg.